src/Simplex/FileTransfer/Types.hs

```diff
@@ -47,6 +47,8 @@ data RcvFile = RcvFile
     key :: C.SbKey,
     nonce :: C.CbNonce,
     chunkSize :: FileSize Word32,
+    redirectEntityId :: Maybe RcvFileId,
```

Comment:
```suggestion
    redirectEntityId :: Maybe DBRcvFileId,
```

?

Reply:
possibly we need both IDs, here and in the database, as we send ByteString to chat probably?